interns-react-weekly-assignments/src/components/Home/Ritika/Week3/utils/validation.ts

```diff
@@ -0,0 +1,23 @@
+import { Employee } from "../interfaces/interface";
+import warnings from "../utils/constants.json"
+
+export const validateInput = (name:string,currentStatus:string) => {
+    if (name.trim() === '' || currentStatus === 'Employee-Status') {
+      alert(warnings.properInputWarning);
+      return false;
+    }
+    return true;
+  };
+  
+export const validationInputAutoFocus=(name:string,phoneNumber:string)=>{
```

Comment:
function names should start with a verb you can change it to validateInputAutoFocus here. Please check across you implementation.